src/Resources/app/storefront/src/checkout/confirm-order.plugin.js

```diff
@@ -0,0 +1,64 @@
+import Plugin from 'src/plugin-system/plugin.class';
+import DomAccess from 'src/helper/dom-access.helper';
+import StoreApiClient from 'src/service/store-api-client.service';
+import FormSerializeUtil from 'src/utility/form/form-serialize.util';
+import ElementLoadingIndicatorUtil from 'src/utility/loading-indicator/element-loading-indicator.util';
+
+/* global adyenCheckoutOptions */
+/* eslint-disable no-unused-vars */
+export default class ConfirmOrderPlugin extends Plugin {
+
+    init() {
+        const confirmOrderForm = DomAccess.querySelector(document, '#confirmOrderForm');
+        confirmOrderForm.addEventListener('submit', this.confirmOrder.bind(this));
+    }
+
+    confirmOrder(event) {
+        event.preventDefault();
```

Comment:
Some defensive programming here because I faced the issue when these vars were not available and then the whole process just died.

```suggestion
        if (!!adyenCheckoutOptions && !!adyenCheckoutOptions.paymentUrl && adyenCheckoutOptions.checkoutOrderUrl) {
        event.preventDefault();
```